sql/testdata/aggregate

```diff
@@ -5,11 +5,15 @@ CREATE TABLE kv (
 )
 
 statement OK
-INSERT INTO kv VALUES (1, 2), (3, 4), (5, NULL)
+INSERT INTO kv VALUES (1, 2), (3, 4), (5, NULL), (6, 2), (7, 2), (8, 4)
 
 query error column "k" must appear in the GROUP BY clause or be used in an aggregate function
 SELECT COUNT(*), k FROM kv
 
+# TODO(pmattis): fix
```

Comment:
I thought it was valuable documentation here especially given that the error message is misleading.